config/locales/server.en.yml

```diff
@@ -1840,6 +1840,7 @@ en:
     min_trust_level_to_allow_profile_background: "The minimum trust level required to upload a profile background"
     min_trust_level_to_allow_user_card_background: "The minimum trust level required to upload a user card background"
     min_trust_level_to_allow_invite: "The minimum trust level required to invite users"
+    min_trust_level_to_ignore: "The minimum trust level required to ignore users"
```

Comment:
I think we should make the name consistent with the other site setting? `min_trust_level_to_ignore` ➡ `min_trust_level_to_allow_ignore`.

Reply:
Sure, works for me!